nibabel/tests/test_arrayproxy.py

```diff
@@ -11,19 +11,24 @@
 from __future__ import division, print_function, absolute_import
 
 import warnings
+import gzip
 
+import pickle
 from io import BytesIO
 from ..tmpdirs import InTemporaryDirectory
 
 import numpy as np
 
-from ..arrayproxy import ArrayProxy, is_proxy, reshape_dataobj
+from ..arrayproxy import (ArrayProxy, KEEP_FILE_OPEN_DEFAULT, is_proxy,
+                          reshape_dataobj)
+from ..openers import ImageOpener
 from ..nifti1 import Nifti1Header
 
 from numpy.testing import assert_array_equal, assert_array_almost_equal
 from nose.tools import (assert_true, assert_false, assert_equal,
                         assert_not_equal, assert_raises)
 from nibabel.testing import VIRAL_MEMMAP
+import mock
```

Comment:
I think this should go above the other less-standard test-related modules like numpy.testing.